addons/website/static/src/scss/website.scss

```diff
@@ -346,7 +346,8 @@ $-seen-urls: ();
 // bootstrap if possible.
 #wrapwrap:not(.o_header_overlay) header, header.o_header_is_scrolled {
     .nav-item > .nav-link > *, .nav-item > .nav-link::after, .js_language_selector span, .badge {
-        color: o-website-value('header-text-color') !important;
+        $-header-text-color: o-website-value('header-text-color');
+        color: o-color(#{$-header-text-color}) or $-header-text-color !important;
```

Comment:
- Why the use of `#{...}`?
- Why the `or` part?

Probably `color: o-color(o-website-value('header-text-color'))` ?

Reply:
> Why the `or` part?

Because if, for example, the color is `#19baba` (a random hexadecimal color not mapped in `$colors`, `$theme-colors` or` $grays`) o-color will return undefined. I could have adapted `o-color` to avoid this `or` but I prefer don't touch it in stable.

> Why the use of #{...}

I think to have double quotes between the param, but I 'll check again

Reply:
> Because if, for example, the color is #19baba o-color will return undefined.

No it does not

> I think to have double quotes between the param, but I 'll check again

Why do you want double quotes? That's why when you give it a color it returns undefined: you don't give it a color but a color converted into a string

Reply:
> That's why when you give it a color it returns undefined: you don't give it a color but a color converted into a string

Indeed but `o-color(800)` returns nothing while `o-color("800")` returns `#343A40`
That's why I needed to do that :thinking: I think that gray color name needs to be a string to be retrieved in the `$grays` map and also not to be converted by `use-cc-bg` in the `o-color` function.

Reply:
Dammit, forgot to send my message yesterday :grimacing: 

Ok, then problem is really that the color is **saved** without quotes. If you choose "o-color-x" as a background of a color preset, you'll see it is saved quoted:
![image](https://github.com/user-attachments/assets/a7646163-0a6c-45d7-9e42-ac993a6e36bd)
But indeed, it's not in the case you are fixing:
![image](https://github.com/user-attachments/assets/cfcc2226-590b-4528-8c0d-cf02fd472abd)

=> That's what we should fix. And in fact, we should not make it so your fix applies automatically for people that saved such a thing :point_up: (keep it not working when it is not quoted). Imagine if the user clicked on "o-color-1", saw no effect... then he receives your fix and their header text becomes red without notice... :grimacing: